docs/en-US/references/xdebug-and-phpstorm.md

```diff
@@ -0,0 +1,65 @@
+# Debugging with xdebug and phpStorm
+
+The basic idea in phpStorm is that you tell it where to look for the xdebug session (the "server") and then you tell it how to link the files it's running in the VM to the files found in the repo (called "mapping").
+
+> What follows is opinionated defaults, so adjust as you deem necessary. Specifically, PHP versions you'll be supporting
+
+## Confirm phpStorm's project setup
+
+Make sure you have your project folders already set up in phpStorm.
+
+1. Click PHPStorm, Preferences, Directories
+2. Make sure your project's content root is there
+3. Click Languages & Frameworks, PHP and fill it in like this
+   - PHP language level: 7.3
```

Comment:
We need to write that is better to align this value with the one inside the `config.yml` for the specific domain. If a custom one is not defined the value is the php default version in the machine that now is php 7.2.